src/entrypoints/my-index.ts

```diff
@@ -106,7 +120,9 @@ class MyIndex extends LitElement {
     this._error = undefined;
 
     if (changeRequestedFromRedirect) {
-      history.back();
+      window.location.replace(
+        `/redirect/${decodeURIComponent(changeRequestedFromRedirect)}`
```

Comment:
```suggestion
        `/redirect/${decodeURIComponent(changeRequestedFromRedirect)}/`
```

Now that we do this, we can also clean up all `changingInstance = true` logic from `my-redirect`.